src/backend/users/views.py

```diff
@@ -106,3 +124,22 @@ def get(self, request, **kwargs):
             "voucherNumber": user_profile["voucher_number"],
         }
         return Response(content)
+
+
+class SignUpPage(APIView):
+    def post(self, request, **kwargs):
+        signup_message = "Thank you! You'll receive an email shortly with a link to complete your account. Click the link to create your profile and get started with ECHO."
+        try:
+            user_serializer = HouseSeekerSignUpSerializer(data=request.data)
```

Comment:
```suggestion
        try:
            with transaction.atomic():
                user_serializer = HouseSeekerSignUpSerializer(data=request.data)
```

I'm realizing after the fact I don't have a good way to rollback any database transactions in the event something happens between User creation --> add to HouseSeekers --> UserProfile creation within the serializer. Does adding something like the above make sense here?

Reply:
Hah, yes! I made a comment about this in another spot before seeing this.

Reply:
Awesome! I see your comment below and will edit this to use the @transaction.atomic decorator